### ISSUE TITLE
vscode-ws-jsonrpc  Do not support cjs？

### ISSUE THREAD

**Issue by NightMingGit** [TypeFox/monaco-languageclient#484]
vscode-ws-jsonrpc  Do not support cjs？

**Comment by NightMingGit** [comment]
i want use in nodejs

**Comment by kaisalmen** [comment]
@NightMingGit no more, please see:
https://github.com/TypeFox/monaco-languageclient#september-2022

**Comment by NightMingGit** [comment]
> @NightMingGit说的不多，请看： https://github.com/TypeFox/monaco-languageclient#september-2022

So I can't use it in a node environment

**Comment by NightMingGit** [comment]
> @NightMingGit说的不多，请看： https://github.com/TypeFox/monaco-languageclient#september-2022

So I can't use it in a node environment

**Comment by CGNonofr** [comment]
You need to use a bundler like rollup to transform the code

**Comment by NightMingGit** [comment]
> @NightMingGit说的不多，请看： https://github.com/TypeFox/monaco-languageclient#september-2022

So I can't use it in a node environment

**Comment by kaisalmen** [comment]
> So I can't use it in a node environment

Yes you cam, if you have the liberty/possibility to use a node environment that supports esm **OR** you transform the code as @CGNonofr suggested.
